L/LibCURL/build_tarballs.jl

```diff
@@ -3,12 +3,14 @@
 using BinaryBuilder
 
 name = "LibCURL"
-version = v"7.68.0"
+version = v"7.70.0"
 
 # Collection of sources required to build LibCURL
 sources = [
-    "https://curl.haxx.se/download/curl-$(version).tar.bz2" =>
-    "207f54917dd6a2dc733065ccf18d61bb5bebeaceb5df49cd9445483e8623eeb9",
+    ArchiveSource("https://curl.haxx.se/download/curl-$(version).tar.gz", 
+    "ca2feeb8ef13368ce5d5e5849a5fd5e2dd4755fecf7d8f0cc94000a4206fb8e7"), 
+    FileSource("https://curl.haxx.se/ca/cacert.pem", 
```

Comment:
Is this file permanent?  Is there a versioned copy?

Reply:
done